containers/proxy-tftpd-image/tftp_wrapper.py

```diff
@@ -80,26 +153,29 @@ def get_response_data(self):
             return HttpResponseData(f"https://{target}/tftp/{path}", capath)
         elif path.startswith("pxelinux.cfg/default"):
             # server local default
-            logging.debug(f"Got request for {path}, forwarding to HTTP")
-            return HttpResponseData(f"https://{target}/tftp/{path}", capath)
+            logging.debug(f"Got request for {path}, filtering HTTP")
+            return HttpResponseDataFilteredPXE(f"https://{target}/tftp/{path}", capath, target)
         elif path.startswith("pxelinux.cfg/"):
             # ignore other pxelinux.cfg files
             logging.debug(f"Got request for {path}, ignoring")
             raise FileNotFoundError()
+        elif path.startswith("grub/x86_64_menu_items.cfg"):
```

Comment:
This is not enough. There are other architectures as well. Especially in a branch, I would expect at least to have ARM and in the future possibly RISC-V present. Power and S390 probably not, however since a proxy can also mean you just have a second datacenter, I wouldn't eliminate the possibility fully.

Reply:
Right. For retail x86_64 and ARM are supported.

Reply:
By the name of the folder, I would suspect that this image is not only used for Retail but for all machines behind a containerized proxy?

Reply:
Yes, but for not matched entries / architectures there is a catch all which returns unfiltered data. We need to filter only retail supported archs.

Reply:
Okay good. Then consider my inquiry satisfied.

Reply:
OK, adding ARM makes sense.